recipe/meta.yaml

```diff
@@ -1,16 +1,37 @@
 {% set name = "usgscsm" %}
-{% set version = "1.7.0" %}
+{% set version = "2.0.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/USGS-Astrogeology/{{ name }}/archive/{{ version }}.tar.gz
-  sha256: 4a60490399c44de5818e4ae49cd941b03b42cf78464fa84fb0a8ed35f52603c5
+  - folder: .
+    url: https://github.com/DOI-USGS/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: a8ae036caa0b958fa6e28ee8ba7a29c2b6fefeb1413144243d59521f51d1dc12
+  
+  # submodules
+  - folder: ale
+    url: https://github.com/DOI-USGS/ale/archive/a7d8a148f54d5141c63ed45a24a71c890ca11de7.zip
+    sha256: d2cecca296de5c44245f0471cf5a3e835fb4792cc9fba24ab9f29822dc67338f
```

Comment:
Once the ALE 0.10.0 release is out, should be in the next 30 minutes, this should be updated to target the 0.10.0 release rather than a commit hash

Reply:
Yes, plan is to set `USGSCSM_EXTERNAL_DEPS=ON` in the build.sh and we can hopefully remove specifying the submodules in meta.yaml.